packages/replay-next/pages/variables.css

```diff
@@ -9,10 +9,10 @@
   --theme-textbox-box-shadow: rgba(97, 181, 255, 0.75);
 
   /* Text sizes */
-  --devtools-base-font-size: 12px;
+  —font-size-regular: 12px;
   --root-font-size: 16px;
-  --theme-body-font-size: 12px;
-  --theme-code-font-size: 11px;
+  --font-size-regular: 12px;
+  --font-size-regular-monospace: 11px;
```

Comment:
These values are already set in the `:root.prefers-default-font-size` and `:root.prefers-large-font-size` blocks:

https://github.com/replayio/devtools/blob/b78b180d8141efc3837975151b086a06fc54a7aa/packages/replay-next/pages/variables.css#L127-L142

I don't think we need to (or should) re-define them here.

```suggestion
```

Reply:
Sounds good - font size is so foundational for a CSS layout that it feels strange having it down on line 127, so I pulled it to the top of the document and removed the dupes.